### PR TITLE
win: 'ffast-math' option is not recognized by clang-cl

### PR DESCRIPTION
**Pull request by kaadam**
Needs to use '/fp:fast' [1] compiler option instead.

[1]: https://docs.microsoft.com/en-us/cpp/build/reference/fp-specify-floating-point-behavior?view=msvc-160#fast

### CODE REVIEW

File: runtime/flang/CMakeLists.txt [hunk 1]
```diff
@@ -560,12 +560,21 @@ set_property(
  PG_PIC
  )
 
-set_source_files_properties(
-  ${I8_FILES_DIR}/red_norm2_stride1.c
-  red_norm2_stride1.c
-  PROPERTIES
-  COMPILE_FLAGS "-ffast-math"
-  )
+if (NOT MSVC)
```
Comment (isuruf) [inline runtime/flang/CMakeLists.txt:563]: To support using clang.exe which has GNU like command line flags.
```suggestion
if (NOT MSVC OR "${CMAKE_C_COMPILER_FRONTEND_VARIANT}" STREQUAL "GNU")
```

Comment (kaadam) [reply runtime/flang/CMakeLists.txt:563]: Yes, you're right. Thanks.

**Comment by kaadam** [comment]
@kiranchandramohan May I ask you to merge this too? Thanks.

**Comment by kaadam** [comment]
@kiranchandramohan may I ask you to merge this PR? It seems it meets with the merge criteria. Thanks.

**Pull request merged**